charts/openforms/values.yaml

```diff
@@ -545,6 +545,15 @@ nginx:
     # requests:
     #   cpu: 250m
     #   memory: 256Mi
+  staticFileServing:
+    # -- Enable static file serving directly from Nginx (requires OpenForms >= 3.3.0)
+    enabled: false
+    # -- Minimum required OpenForms version for static file serving (semver format)
+    minVersion: "3.3.0"
+    # -- Initial delay seconds for liveness probe when static file serving is enabled
+    livenessProbeInitialDelaySeconds: 180
+    # -- Initial delay seconds for readiness probe when static file serving is enabled
+    readinessProbeInitialDelaySeconds: 120
```

Comment:
isn't that like super long?

I understand that the OF container image needs to be pulled, start up and copy the assets to the volume which can take some time, but 2 minutes sounds excessive.

Additionally, the volume mount will be present, just not fully populated yet (at worst), but I have the impression now that it will take at least 2 minutes before the container will serve traffic which sounds like a long delay.

References to the new static files will only be produced by the new application container anyway, so even if the assets aren't fully copied yet, there will never be a request to one of the new assets because no application container is live yet.